.github/workflows/ci.yaml

```diff
@@ -65,19 +65,18 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.8', '3.9', '3.10', '3.11']
         os: [ubuntu, macos, windows]
         cmd: [sdk, cli]
         exclude:
-        - os: macos  # temporarily exclude
-        - python-version: '3.8'
-          os: macos
-        - python-version: '3.8'
-          os: windows
         - python-version: '3.9'
           os: macos
         - python-version: '3.9'
           os: windows
+        - python-version: '3.10'
+          os: macos
+        - python-version: '3.10'
+          os: ubuntu
```

Comment:
why ubuntu here?

Reply:
The logic is as follows:
- test for all OSes on edge versions (latest and earliest available).
- test for a single OS for inner versions. Here I just picked ubuntu for "3.9" and Windows for "3.10" (mostly randomly, to be honest).